Sources/MapboxNavigation/UserPuckStyleKit.swift

```diff
@@ -0,0 +1,132 @@
+//
+//  UserPuckStyleKit.swift
+//  MapboxNavigation
+//
+//  Created on 2017-10-11.
+//  Copyright © 2017 Mapbox. All rights reserved.
+//
+//  Generated by PaintCode
+//  http://www.paintcodeapp.com
+//
+
+import UIKit
+
+public class UserPuckStyleKit : NSObject {
```

Comment:
Should it be public?

Reply:
The `ManeuversStyleKit`, `SpeedLimitStyleKit` and `LanesStyleKit` are all public now. But because the `UserPuckStyleKit` right now is only used to draw the `UserPuckCourseView`. And we could customize it through the  `UserPuckCourseView`, I'll remove its public exposure.